src/test/resources/pages/instructorFeedbackRubricQuestionAddSuccess.html

```diff
@@ -876,7 +876,7 @@ <h5 class="col-sm-2">
                             </span>
                           </span>
                           <textarea class="form-control" disabled="" id="rubricSubQn-1-1" name="rubricSubQn-1" required="" rows="3">
-                            This student completes assigned tasks on time.
+                            sub question text
```

Comment:
I'm wondering what caused this change? `sub question text` is given only in the `empty sub question test` so it shouldn't appear here as a change?

Reply:
Actually, it is because of how the whole test class is arranged.

Earlier, what was happening is, after the `empty sub question test`, one of the sub question had `sub question text` as text. (as, empty sub question is checked at the front-end, the question form remains with all it's changes even if we fail the validation).

 this line was getting removed at the `empty weight test` as previously there was no front-end validation for empty weight, when the backend validation of weight used to fail in the `empty weight test`, the whole question frame was lost due to the redirect.

After that the `testAddQuestionAction()` used to add a whole new question with default values. 

Now, as the weight is tested in the front-end, like the sub questions, the question form remains even after the validation test. SO, I cancel the question frame after the test is over. Which means, when `testAddQuestionAction()` adds a new rubric question, as the previous form was cancelled, the `sub question text` remains.

Reply:
I see 👍